doc/sphinx/source/tutorials/apfelcomb.rst

```diff
@@ -0,0 +1,362 @@
+
+If the new dataset is a compound observable (that is, theory predictions
+are a function of more than one FK-product), then one should write a
+corresponding ``COMPOUND`` file as described in `Theory data
+files <../data/th-data-files.rst#compound-file-format>`__. This compound
+file should be stored in the APFELcomb repository under the ``compound``
+directory.
+
+C-factors should be in the format specified in `Theory data
+files <../data/th-data-files.rst#cfactor-file-format>`__ and stored in
+the nnpdfcpp repository under
+
+::
+
+   nnpdf/nnpdfcpp/data/N*LOCFAC/
+
+directory.
+
+Important note on subgrid ordering
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If the FK table consists of more than one subgrid to be merged into a
+single table, then the ordering of the subgrids in their subgrid **id**
+is vital. The ``merge_allgrids.py`` script will merge the subgrids in
+order of their **id**. So if one is constructing an FK table for a
+merged W+/W-/Z dataset, it is crucial that the ordering of the
+corresponding W+/W-/Z subgrids in id matches the ordering in
+``COMMONDATA``.
+
+Important note on committing changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If one makes a modification to the ``apfelcomb.db`` database, once he is
+happy with it one *must* export it to the plain-text dump file at
+``db/apfelcomb.dat``. This file must then be committed. It is important
+to note that the binary sqlite database is not stored in the repository.
+
+A helper script is provided to do this. If you want to convert your
+binary database to the text dump, run ``db/generate_dump.sh`` and then
+commit the resulting ``apfelcomb.dat`` file.
+
+Also, note that, if one conversely modifies the ``apfelcomb.dat`` file,
+one has to delete and re-generate the sqlite database ``apfelcomb.db``
+This is easily done by running ``db/generate_database.sh``.
+
+Helper scripts
+--------------
+
+Several helper scripts are provided to make using APFELcomb easier
+(particularly when generating a full set of FK tables for a particular
+theory). - ``scripts/disp_grids.py`` displays a full list of
+APPLgrid/FastNLO, DIS or DYP subgrids implemented in APFELcomb. -
+``run_allgrids.py [theoryID] [job script]`` scans the results directory
+and submits jobs for all missing subgrids for the specified theory. -
+``test_submit.py`` is an example [job script] to be used for
+``run\_allgrids.py``. These scripts specify how jobs are launched on a
+given cluster. - ``hydra_submit.py`` is the [job script] for the HYDRA
+cluster in Oxford. - ``merge_allgrids.py [theoryID]`` merges all
+subgrids in the results directory for a specified theory into final FK
+tables. This does not delete subgrids. - ``finalise.sh [theoryID]`` runs
+C-factor scaling, copies ``COMPOUND`` files, deletes the subgrids, and
+finally compresses the result into a theory.tgz file ready for upload. -
+``results/upload_theories`` automatically upload to the server all the
+theory.tgz files that have been generated.
```

Comment:
It seems to be in general for this file at least that the lists have not been parsed correctly.